docs/developer/guidelines/style.md

```diff
@@ -46,4 +46,51 @@ A `flake8` plugin for ensuring a consistent logging format. We enable:
 
 ## [Mypy][mypy-github]
 
-A type checker allowing a mix of dynamic and static typing. This is optional for now.
+A comment-based type checker allowing a mix of dynamic and static typing. This is optional for now. In order to enable `mypy` for a specific integration, open its `tox.ini` file and add the 2 lines in the correct section:
+
+```
+[testenv]
+dd_check_types = true
+dd_mypy_args = <FLAGS> datadog_checks/ tests/
+...
+```
+
+The `dd_mypy_args` can be used to add or overwrite some flags to mypy for this specific integration. Here are some useful arguments you can add:
```

Comment:
Another useful flag is [`--check-untyped-defs`](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-check-untyped-defs), which allows mypy to check the inside of functions even if signatures are not annotated (thus treating any unannotated argument as any).

(I've been thinking we might want to switch to this mode by default, but we owe us a type hints retro at some point so that's the kind of thing we'll discuss there. Still, interesting flag to mention IMO.)